docs/content/guide/filter.ngdoc

```diff
@@ -109,7 +109,7 @@ as the first argument. Any filter arguments are passed in as additional argument
 function.
 
 The filter function should be a [pure function](http://en.wikipedia.org/wiki/Pure_function), which
-means that it should be stateless and idempotent, and not rely for example on other Angular services.
+means that it should be deterministic and not rely, for example, on other Angular services.
```

Comment:
Why remove `idempotent`? It is a very important characteristic and is not synonym to deterministic.
Also, "stateless" (even if not 100% accurate) is important (and again not covered by deterministic).